build.sbt

```diff
@@ -269,6 +270,29 @@ lazy val solrZio2 =
       )
     )
 
+lazy val localstackZio2 =
+  project
+    .in(file("modules/localstack-zio-2.0"))
+    .settings(
+      zioSeries := ZIOSeries.Series2X,
+      testcontainersScalaSettings,
+      name := "zio-2.0-testcontainers-localstack",
+      libraryDependencies ++= Seq(
+        "com.dimafeng" %% "testcontainers-scala-localstack" % V.testcontainersScalaVersion,
+        "com.amazonaws" % "aws-java-sdk-s3"                 % V.awsV1Version  % Provided,
+        "com.amazonaws" % "aws-java-sdk-sqs"                % V.awsV1Version  % Provided,
```

Comment:
That's the way `testcontainers-scala` provides these dependencies. I decided to stick to their approach.